### DISCUSSION TITLE
1.5.1.1

### DISCUSSION THREAD

**Discussion by ilyhalight** [ilyhalight/voice-over-translation#553] [Обновления]
## Список изменений
- Исправление работы при отключенном автовоспроизведение (#551) от @SashaXser в #552
- Добавлена библиотека `npm-run-all` для ускорения процесса сборки с помощью параллельного запуска команд. Благодаря этому полная сборка расширения была ускорена в ~2 раза.

## What's Changed
* Fix #551 by @SashaXser in https://github.com/ilyhalight/voice-over-translation/pull/552

**Full Changelog**: https://github.com/ilyhalight/voice-over-translation/compare/1.5.1...1.5.1.1

<hr /><em>This discussion was created from the release <a href='https://github.com/ilyhalight/voice-over-translation/releases/tag/1.5.1.1'>78c3786d91f87742830bc5f402799eed89225545</a>.</em>

**Comment by FiLinX** [comment]
странно, я вот тоже много лет уж пользую Enhancer for YouTube с блоком автоплея там, и к тому же я почти все ютубины отправляю открываться в фоновые вкладки (набиваю так для просмотра заинтересовавшие новинки с ленты подписок), а потом их смотрю.
так вот у меня панелька перевода на этих в фоне открытых и стоящих на паузе ютубинах - присутствует на 1.5.1
но увы она отсутствует на embed линках(
но это уже другая тема, ни к Enhancer for YouTube ни к в фоне открытию не относиться

**Reply by NullVerdict** [reply]
[Попробуй](https://github.com/SashaXser/voice-over-translation/tree/main/dist), исправил баг с embed.

**Comment by NullVerdict** [comment]
> странно, я вот тоже много лет уж пользую Enhancer for YouTube с блоком автоплея там, и к тому же я почти все ютубины отправляю открываться в фоновые вкладки (набиваю так для просмотра заинтересовавшие новинки с ленты подписок), а потом их смотрю. так вот у меня панелька перевода на этих в фоне открытых и стоящих на паузе ютубинах - присутствует на 1.5.1 но увы она отсутствует на embed линках( но это уже другая тема, ни к Enhancer for YouTube ни к в фоне открытию не относиться

Заметил проблему, сейчас отправлю исправление.
